server/src/main/java/io/spine/server/ServerEnvironment.java

```diff
@@ -393,14 +400,20 @@ public StorageFactory storageFactory() {
     public TransportFactory transportFactory() {
         Class<? extends EnvironmentType> type = environment().type();
         TransportFactory result = transportFactory.optionalValue(type)
-                .orElseThrow(() -> newIllegalStateException(
-                        "Transport factory is not assigned for the current environment `%s`."
-                                + " Please call `.when(environmentType).use(transportFactory)`.",
-                        type.getSimpleName()));
+                .orElseThrow(() -> suggestConfiguringTransportFactory(type));
 
         return result;
     }
 
+    private static IllegalStateException
+    suggestConfiguringTransportFactory(Class<? extends EnvironmentType> type) {
```

Comment:
Should we have the same methods for the tracer factory and delivery?

If so, maybe we should arrange a private nested class `Suggest` with the names like `configuringTransportFactory`?

Reply:
Or event `SuggestConfiguring.transportFactory(..)`?